oar/cli/cmd_drop_bugs.py

```diff
@@ -34,6 +35,22 @@ def drop_bugs(ctx):
             NotificationManager(cs).share_dropped_and_must_verify_bugs(
                 dropped_bugs, must_verify_bugs
             )
+            requested_doc_ads=[]
+            requested_prodsec_ads=[]
+            if len(approved_doc_ads):
+                for ad in approved_doc_ads:
+                    ad.refresh()
+                    if not ad.is_doc_approved() and  not ad.is_doc_requested():
+                        ad.request_doc_approval()
+                        requested_doc_ads.append(ad)
+            if len(approved_prodsec_ads):
+                for ad in approved_prodsec_ads:
+                    ad.refresh()
+                    if not ad.is_prodsec_approved() and ad.is_prodsec_requested() == 'null':
+                        ad.request_prodsec_approval()
```

Comment:
changes from advisory mgr is missed here

Reply:
sorry, what do you mean here?

Reply:
`ad.request_prodsec_approval()` is code in advisory mgr

Reply:
ok, checked the latest code, the func name is ok,